src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -17,6 +17,7 @@ import {
   SadSmileyIcon,
 } from '../../../assets/images'
 import Button from '../../components/Button'
+import PressableOpacity from '../../components/PressableOpacity'
```

Comment:
When on exerciseFinishedScreen, and clicking the close-icon in the header it gets a background and looks weird, it should just change color of the text/icon, e.g. like the qr-code-scanner icon.

Reply:
removed bg-color for `PressableOpacity` and it should be fine now. So bg-color changes only happen with `Button`component